firefox-ios/Storage/Rust/RustFxAKeychain.swift

```diff
@@ -0,0 +1,16 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Glean
+import Shared
+import Common
+
+public class RustFxAKeychain: @unchecked Sendable {
+    public static func reportFxaKeychainUsageTelemetry() {
+        DispatchQueue.global().async {
+            GleanMetrics.FxaKeychainUsage.enabled.record()
+        }
```

Comment:
@lougeniaC64 I don't think we need the async queue here unless there was a good reason for it ?
```suggestion
        GleanMetrics.FxaKeychainUsage.enabled.record()
```

Reply:
Nope, let's nuke it!